t/cache.t

```diff
@@ -79,7 +77,6 @@ isnt $new_cache, $cache, "Got new cache object";
 
 ok(Cache::RedisDB->set("Test", "TTL", "I will expire", 60), "Set TTL test key for 60 second expiration.");
 is(Cache::RedisDB->get("Test", "key1"), "value1", "Got value1 for Test::key1");
-is(Cache::RedisDB->ttl("Test", "TTL"), 59, "A moment later the expiration of TTL is down to 59 seconds.");
```

Comment:
I originally switched it to `60`, but the test removed here already exists: see L115.